src/view/delete/delete-event-modal.tsx

```diff
@@ -19,23 +19,25 @@ const ItalicText = styled(Text)`
 `;
 
 // eslint-disable-next-line @typescript-eslint/explicit-module-boundary-types
-export const DeleteEventModal = ({ event, open, isInstance, onClose }: any): ReactElement => {
+export const DeleteEventModal = ({ event, onClose }: any): ReactElement => {
+	const isInstance = !!event?.resource?.ridZ;
 	const invite = useInvite(event?.resource?.inviteId);
 	const [t] = useTranslation();
 	const [isAskingConfirmation, setIsAskingConfirmation] = useState(false);
-
+	const replaceHistory = useReplaceHistoryCallback();
 	const toggleAskConfirmation = useCallback(() => {
 		setIsAskingConfirmation((a) => !a);
 	}, []);
 
 	const actions = useDeleteActions(event, invite, {
 		isInstance,
+		replaceHistory,
 		onClose
 	});
 
 	return (
 		<CustomModal
-			open={open}
+			open
```

Comment:
why always open ? and if always need to be open then why onClose function to close it ?